FBRetainCycleDetector/Detector/FBRetainCycleDetector.mm

```diff
@@ -112,31 +112,33 @@ - (void)addCandidate:(id)candidate
     @autoreleasepool {
       // Take topmost node in stack and mark it as visited
       FBNodeEnumerator *top = [stack lastObject];
+      // Take object that topmost node monitor
+      __weak id object = top.object.object;
```

Comment:
Maybe it needs to mimic [`storeWeak`](https://github.com/zhongwuzw/objc4-cn/blob/master/runtime/objc-weak.mm#L393-L418)?